reference_to_manus:
  x: 0.2
  y: 0.0
  z: 0.0
  qx: 0.0
  qy: 0.0
  qz: 0.0
  qw: 1.0
mode: "IK" # "IK" | "Scaled"
scaling_factor: 
  # x: 1.4
  # y: 1.4
  # z: 1.3
  x: 1.7
  y: 1.4
  z: 1.55

tuning_mode: false # Set this to true while tuning scaling_factor and 